src/main/resources/reference.conf

```diff
@@ -143,4 +143,2022 @@ akka {
   http {
     server.request-timeout = 1 minute
   }
-}
\ No newline at end of file
+}
+
+# This is the configuration for metadata overrides. You can override visibility and dislpay name for platform, network,
+# entities and attributes. On each level you can set two parameters:
+# * display-name: String
+# * visible: Boolean (false by default
+
+metadata-overrides {
+  tezos {
+    visible: true
+    networks {
+      mainnet {
+        visible: true,
+        entities {
+          accounts {
+            visible: true,
+            attributes {
+              account_id {
+                visible: true
```

Comment:
Missing `description` field as specified in #316

Reply:
This PR is already approved, so I added this to https://github.com/Cryptonomic/Conseil/pull/344 to simplify code review.